File: Python/2908번.py
a,b = map(str,input().split())
p = a[-1]
q = b[-1]
for i in range(len(a)-2,-1,-1):
    p += a[i]
    q += b[i]

if int(p) < int(q):
    print(q)
else:
    print(p)

File: Java/2908번.java
package Java_Algorithm;

import java.util.Scanner;

public class Const {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        StringBuffer p = new StringBuffer(input.next());
        StringBuffer q = new StringBuffer(input.next());

        p.reverse();
        q.reverse();

        for (int i = 0; i < p.length(); ++i) {
            if ((int)(p.charAt(i)) < (int)(q.charAt(i))) {
                System.out.println(q.toString());
                break;
            }

            else if ((int)(p.charAt(i)) > (int)(q.charAt(i))) {
                System.out.println(p.toString());
                break;
            }

            else {
                continue;
            }
        }


    }
}
